kanidm_proto/src/v1.rs

```diff
@@ -324,7 +324,7 @@ impl fmt::Display for AuthType {
 ///
 /// It's likely that this must have a relationship to the server's user structure
 /// and to the Entry so that filters or access controls can be applied.
-#[derive(Debug, Serialize, Deserialize, Clone)]
+#[derive(Debug, Serialize, Deserialize, Clone, PartialEq, Eq)]
```

Comment:
Why does this need eq?

Reply:
We need this since we pass the `WhoamiResponse` as a property now (to enable conditional display of the dialog)

I'm not too sure of the implications but if you'd like this to stay the same, we can always add a new struct for the property (which will become obsolete once better state management is implemented in the UI)

Reply:
I think it's okay, I was just curious.